ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml

```diff
@@ -1,14 +1,37 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.13:
```

Comment:
the ci/art guys have been tweaking our existing stuff in real time @coreydaley ... with samples there was an update from them that landed just yesterday that modifies the boilerplate some

https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master.yaml

In particular https://github.com/openshift/release/commit/cfc40d2b8a983969dfe0099bbe7dceac67af1fdb#diff-178e9aea941cbd96467f108576c97a5d

How about you switch to the "builder" naming convention like they have there.  Otherwise what you have now seems pretty close.  And the use of `dockerfile_path` still makes sense to me ... unless we want to move the Dockerfile in the repo up to the root now that we only have one image.

Reply:
pushed an update with the requested changes, and that references the Dockerfile being in the root of the projected-resource-csi-driver repository (after https://github.com/openshift/projected-resource-csi-driver/pull/9 merges)